### PR TITLE
Add context help to native new project wizard

### PR DESCRIPTION
**Pull request by chanseokoh**
Pat of #696. (Takes care of the native wizard.)

new app engine standard project _004

I found that the description in the `context.xml` file honors some HTML tags (e.g., `<b>`) but not every (e.g., `<p>` doesn't work.)

FYI, the `context.xml` file can be placed in/loaded from any bundle, so if it's desired, we can move it to another bundle.

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/build.properties [hunk 1]
```diff
@@ -3,6 +3,7 @@ output.. = target/classes/
 bin.includes = META-INF/,\
                .,\
                plugin.properties,\
-               plugin.xml
+               plugin.xml,\
+               help-contexts/
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/build.properties]: Eclipse plugins typically put this file at the root and call it `helpContexts.xml`.

Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/build.properties]: Okay, I'll move the file.

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/help-contexts/contexts.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<contexts>
+  <context id="NewProjectContext" title="Create App Engine Standard Project">
+     <description>You can create a new Eclipse project that can be deployed to App Engine Standard environment on Google Cloud Platform.
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/help-contexts/contexts.xml]: delete "You can"
delete "on Google Cloud Platform"

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/help-contexts/contexts.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+&lt;b&gt;Project name:&lt;/b&gt; Specify an Eclipse project name.
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/help-contexts/contexts.xml]: Why the &lt;&gt; here?

Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/help-contexts/contexts.xml]: delete "Specify"

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/help-contexts/contexts.xml [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+&lt;b&gt;Add Libraries to build path:&lt;/b&gt; You can add some popular libraries on the App Engine Standard environment to your build path.</description>
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/help-contexts/contexts.xml]: delete "You can"
delete "popular"

Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/help-contexts/contexts.xml]: "Libraries to add to build paht" I.e. the same string as in the UI

**Comment by chanseokoh** [comment]
selection_008 ?

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/help-contexts/contexts.xml (hunk 4 above)
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/help-contexts/contexts.xml]: Oops, good catch.

**Comment by briandealwis** [comment]
It might be worth looking at the help text on the WTP Dynamic Web Project.

**Comment by chanseokoh** [comment]
Yeah, I think we will keep iterating over the text before GA.

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/helpContexts.xml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<contexts>
+  <context id="NewProjectContext" title="Create App Engine Standard Project">
+     <description>Create a new Eclipse project that can be deployed to App Engine Standard environment.
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/helpContexts.xml]: to the App Engine

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/helpContexts.xml [hunk 6]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <contexts>
   <context id="NewProjectContext" title="Create App Engine Standard Project">
-     <description>Create a new Eclipse project that can be deployed to App Engine Standard environment.
+     <description>Create a new Eclipse project that can be deployed to the App Engine.
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/helpContexts.xml]: You need "Standard environment"

**Pull request merged**